schema/tests/Provider/Company/CompanyRepositoryTest.php

```diff
@@ -21,8 +21,23 @@ public function test_runner()
         $this->it_finds_by_brandId();
         $this->it_finds_prepaid_companies();
         $this->it_finds_vpbx_company_ids_by_brand();
+        $this->it_finds_one_by_domain_users();
     }
 
+    public function it_finds_one_by_domain_users(){
```

Comment:
Either call this method `public function it_finds_one_by_domain() {` or call repository method `findOneByDomainUsers`